README.md

```diff
@@ -28,17 +28,15 @@ Gupax may not run on machines with:
 * [License](#license) 
 
 ## What is Gupaxx/XMRvsBeast?
-[**`Gupaxx`**](https://getmonero.org) is a fork of [*Gupax*](https://github.com/hinto-janai/gupax) that integrate the [XMRvsBeast raffle](https://xmrvsbeast.com).
+[**`Gupaxx`**](https://getmonero.org) is a fork of [*Gupax*](https://github.com/hinto-janai/gupax) that integrates the [XMRvsBeast raffle](https://xmrvsbeast.com).
 
-This fork adds an easy way to distribute your hashrate between P2pool and XMRvsBeast to participate in the raffle to win more xmr while supporting the p2pool network.
+With this fork, you can easily split your hashrate between P2Pool and XMRvsBeast, increasing your chances of winning in the raffle while also supporting the P2Pool ecosystem.
 
-To have a detailed explication of Gupax, see the [README](https://github.com/hinto-janai/gupax) of upstream.
+For a explanation of Gupax, see the [README](https://github.com/hinto-janai/gupax) of upstream.
```

Comment:
Why not "For a detailed explanation" ?
and if without "detailed", should not it be "an" instead of "a" ?

Reply:
changed:
For a explanation of Gupax
to:
For an detailed explanation of Gupax

Reply:
> changed: For a explanation of Gupax to: For an detailed explanation of Gupax

changed an to a